README.md

```diff
@@ -98,144 +95,123 @@ different element _lifecycle events_:
+(instance) methods:
 
-1. `createdCallback()` is called when the element is created.
-1. `attachedCallback()` is called whenever the element is added to
-   the document, either directly or indirectly.
-1. `detachedCallback()` is called whenever the element is removed from
+1. `connectedCallback()` is called whenever the element is added to
+   the document, either directly (`document.body.appendChild(el)`) or indirectly
+   (as part of a fragment or a DOM tree that's added to the document).
+1. `disconnectedCallback()` is called whenever the element is removed from
    the document.
 1. `attributeChangedCallback(attr, oldValue, newValue)` is called when
    an _observed_ attribute (see below) is changed.
 
 and the following static (class) properties:
 
-1. `extends` denotes the native HTML element that your custom element
-   will extend via the HTML `is` attribute. See
-   [type extension](#type-extension) for more info.
 1. `observedAttributes` is an optional array of attribute names for
    which the `attributeChangedCallback()` will be called. If you do not
    provide this property, the callback will fire for all attributes.
 
   ```js
   // ES5
-  var CounterElement = {
-    observedAttributes: ['value'],
-    prototype: Object.create(
-      HTMLElement.prototype,
-      {
-        attributeChangedCallback: {value: function(name, old, value) {
-          // we can safely ignore name here because 'value' is the only
-          // observed attribute
-          this.value = value;
-        }},
+  class CounterElement extends HTMLElement {
+    static get observedAttributes() { return ['value'] }
+
+    attributeChangedCallback(name, old, value) {
+      // we can safely ignore name here because 'value' is the only
+      // observed attribute
+      this.value = value
+    }
         
-        value: {
-          get: function() {
-            return ('_value' in this)
-              ? this._value
-              : (this._value = 0);
-          },
-          set: function(value) {
-            this._value = value;
-          }
-        }
-      }
-    )
-  };
+    get value() {
+      return ('_value' in this)
+        ? this._value
+        : (this._value = 0)
+    }
+    
+    set value(value) {
+      this._value = value
+    }
+  }
   ```
 
-## Type Extension
-Custom elements that extend built-in HTML elements with special
-semantics or behaviors (such as `<button>` or `<input>`) must use a
-feature called _type extension_ via the HTML `is` attribute:
+## Customized built-in elements
 
-```html
-<button is="fancy-button">I am fancy</button>
-<!-- this will NOT inherit button's semantics or behavior: -->
-<fancy-button>I am not fancy</fancy-button>
-```
+⚠️ **Warning:** Safari does not implement [this portion](https://html.spec.whatwg.org/multipage/custom-elements.html#customized-built-in-element) of the [spec]. If you wish to use it, you will need a [polyfill](#polyfills).
```

Comment:
> Safari does not _plan to_ implement…

Might be a subtle but important distinction. We don't extend built-in elements with `is` at all for this reason.